Features/Lib/PlatformUI.js

```diff
@@ -0,0 +1,407 @@
+/*
+ * File containing all Javascript implentations for the PlatformUI BDD
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+/**
+ * Container for Javascript functions
+ *
+ * @returns {PlatformUIHelper}
+ */
+function PlatformUIHelper(){
+    //YUI global object with node-event-simulate module
+    var Y = YUI().use('node-event-simulate', 'stylesheet');
```

Comment:
This is wrong, sometimes fails and makes the test unreliable because the YUI module loading is an asynchronous process so it happens that those 2 modules are not loaded while we try to use what they provide.

But in any case, I find weird to rely on YUI in the test and hopefully, this complete file can be removed.